--- Xbae-4.50.2/examples/draw/Makefile.am.orig	2002-08-20 18:21:09.000000000 +0200
+++ Xbae-4.50.2/examples/draw/Makefile.am	2003-12-29 17:37:00.292220752 +0100
@@ -16,5 +16,5 @@
 endif
 
 LDADD=	../../src/libXbae.la \
-        $(link_motif) @X_LIBS@ $(XMU) -lXt ${X_PRE_LIBS} -lXext -lX11 ${X_EXTRA_LIBS}
+        $(link_motif) $(XPM_LIB) @X_LIBS@ $(XMU) -lXt ${X_PRE_LIBS} -lXext -lX11 ${X_EXTRA_LIBS}
 endif
--- Xbae-4.50.2/configure.in.orig	2003-03-12 20:55:48.000000000 +0100
+++ Xbae-4.50.2/configure.in	2003-12-29 17:36:27.704174888 +0100
@@ -165,8 +165,9 @@
 dnl
 if test "x$ac_cv_header_X11_xpm_h" = "xyes" -o "x$ac_cv_header_xpm_h" = "xyes"
 then
-       AC_CHECK_LIB(Xpm, XpmLibraryVersion, , , 
+       AC_CHECK_LIB(Xpm, XpmLibraryVersion, [XPM_LIB="-lXpm"], [XPM_LIB=], 
                     $X_LIBS $X_PRE_LIBS -lX11 $X_EXTRA_LIBS)
+	AC_SUBST(XPM_LIB)
 	AM_CONDITIONAL(HaveXpm, test x$ac_cv_lib_Xpm_XpmLibraryVersion = xyes)
 else
          AM_CONDITIONAL(HaveXpm, false)
